app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
As for documentation at  https://diaspora.github.io/diaspora_federation, we don't have any documentation there for the changes in this PR. Do you want me to add any documentation on our new JSON features within this PR?

> About app/schemas: Is that the correct place to put schemas?
We can move it to `lib/diaspora_federation/schemas` I think.

> Also: In which gem should we package the schema? Should we add it to diaspora_federation? Or we can already create a separate gem for the schema diaspora_federation-json_schema, but release it together with everything else (like we already do for -rails and -test)?

Hmm, why not, let's make a separate gem.